packages/math-input/src/components/touchable-keypad-button.tsx

```diff
@@ -128,7 +138,7 @@ const mapStateToProps = (state, ownProps) => {
 
         // Add in some gesture state.
         focused: gestures.focus === id,
-        popoverEnabled: gestures.popover && gestures.popover.parentId === id,
+        popoverEnabled: !!(gestures.popover?.parentId === id),
```

Comment:
**question:** is `!!` necessary here?

Reply:
Yep, you're right! I totally spaced on this one, thanks for the correction.